bench/mjolnir/buildtiles.cc

```diff
@@ -0,0 +1,55 @@
+#include <benchmark/benchmark.h>
+
+#include "baldr/rapidjson_utils.h"
+#include "filesystem.h"
+#include "mjolnir/util.h"
+
+using namespace valhalla::mjolnir;
+
+namespace {
+
+void BM_SomeFunction(benchmark::State& state, const std::string& pbf_path) {
```

Comment:
This can be named something more appropriate :) Like `BM_BuildTiles` or some such.

Reply:
Ooops, was opypasted from the tutorials I guess. Thx :)